Week-6/Problem_6.3/sort.js

```diff
@@ -0,0 +1,24 @@
+function sortArray(arr) {
+  initialization;
+  let length = arr.length;
+  let num0 = [];
+  let num1 = [];
+  let num2 = [];
+
+  for (let i = 0; i <= length; i++) {
+    if (arr[i] === 0) {
+      num0.push(0);
+    } else if (arr[i] === 1) {
+      num1.push(1);
+    } else if (arr[i] === 2) {
+      num2.push(2);
+    }
+  }
+  // concatinating arrays
+  return [...num0, ...num1, ...num2];
+}
```

Comment:
Need not require to use 3 arrays, it can be solved using counters or using a resultant array to which you can push it front end and rear end , number of 2's can input arr length - resultant array length